app/views/layouts/megatron/application.html.slim

```diff
@@ -8,10 +8,12 @@ html lang="en"
     meta name="viewport" content="width=device-width,initial-scale=1"
     meta charset="utf-8"
     meta http-equiv="X-UA-Compatible" content="IE=edge"
+    meta name="bugsnag" data-api-key="#{ENV['BUGSNAG_API_KEY']}" data-release-stage="#{Rails.env}"
```

Comment:
This makes sense. I guess I was still thinking old-school where the API was baked in. Of course this will only happen in production. I can see it being nice to track things in staging though. Some parts of our apps can only be tested in staging and having bug snagging there seems useful. I guess it can just dump into the same hole as production since it tells which url it comes from anyway.